packages/sync-repo-settings/src/sync-repo-settings.ts

```diff
@@ -150,7 +165,7 @@ export function handler(app: Application) {
     }
   );
 
-  app.on(['schedule.repository'], async (context: Context) => {
+  app.on(['schedule.repository' as any], async (context: Context) => {
```

Comment:
Let's please treat compiler warnings as errors.  Anytime we see one of these, we should mitigate the warning.  In this case - instead of `any`, I think you cast `as '*'` :)